apps/web/src/pages/EventPlanPage/EventPlanning.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import { Row, Col, Container, Modal, Button } from 'react-bootstrap';
+import { Modal } from 'react-bootstrap';
```

Comment:
We are still using bootstrap modal? Or were we going to switch to tailwind

Reply:
I thought so as well, but the creators of those pages kept some aspects I guess